esx_service/utils/auth_data.py

```diff
@@ -1157,6 +1157,27 @@ def remove_volumes_from_volumes_table(self, tenant_id):
 
         return None
 
+    def remove_symlink_for_tenant(self, tenant_id):
+        """
+            Delete the symlink /vmfs/volume/datastore_name/tenant_name
+            which point to /vmfs/volumes/datastore_name/tenant_uuid
+        """
+        err, tenant_name = self.get_tenant_name(tenant_id)
+        if err:
+            return err
+        VOL_RM_LOG_PREFIX = "Tenant <name> %s removal: "
+        for (datastore, _, path) in vmdk_utils.get_datastores():
```

Comment:
Does this have to be a loop over all datastores while wanting to remove only the symlink for a specific datastore? Can this loop over instead on just the datastores assigned to the vmgroup.

What happens if there are multiple datastores assigned to a single tenant, that is also a test case and should also be verified.